src/html/html-script-tag.ts

```diff
@@ -27,7 +27,10 @@ export class ScriptTagImport extends Import { readonly type = 'html-script'; }
  * A synthetic import that provides the document containing the script tag to
  * the javascript document defined/referenced by the script tag.
  */
+// TODO(usergenic): Reconsider whether this should behave like the
```

Comment:
Let's file issues for TODOs, and reference the issue.

Reply:
Done.  Also removed the TODO about reconsidering behavior.  I considered it and decided it makes sense as an Import.